xbmc/video/VideoInfoTag.cpp

```diff
@@ -477,6 +484,7 @@ void CVideoInfoTag::Serialize(CVariant& value) const
   value["seasonid"] = m_iIdSeason;
   value["specialsortseason"] = m_iSpecialSortSeason;
   value["specialsortepisode"] = m_iSpecialSortEpisode;
+  value["specialflag"] = m_iSpecialFlag;
```

Comment:
This would also benefit from a string-value as the enum value might change due to re-ordering or whatever and would then break the API (if it is fully exposed to JSON-RPC).